Python build error lldb-dap ubuntu 26.04

Can't build `lldb-dap` on Ubuntu 26.04 (x86_64 WSL2), I got this python-related error below:

```
$> cmake --build build --target lldb-dap
[1/67] Running utility command for liblldb-header-staging
[2/67] Building Options.inc...
[3/67] Building CXX object tools/lldb/tools/lldb-dap/CMakeFiles/lldbDAP.dir/InstructionBreakpoint.cpp.o
[4/67] Building CXX object tools/lldb/tools/lldb-dap/CMakeFiles/lldbDAP.dir/ExceptionBreakpoint.cpp.o
[5/67] Building CXX object tools/lldb/tools/lldb-dap/CMakeFiles/lldbDAP.dir/FunctionBreakpoint.cpp.o
[6/67] Building CXX object tools/lldb/tools/lldb-dap/CMakeFiles/lldbDAP.dir/FifoFiles.cpp.o
[7/67] Building CXX object tools/lldb/tools/lldb-dap/CMakeFiles/lldbDAP.dir/LLDBUtils.cpp.o
[8/67] Building CXX object tools/lldb/source/API/CMakeFiles/liblldb.dir/__/__/bindings/python/LLDBWrapPython.cpp.o
FAILED: tools/lldb/source/API/CMakeFiles/liblldb.dir/__/__/bindings/python/LLDBWrapPython.cpp.o 
/usr/bin/c++ -DGTEST_HAS_RTTI=0 -DHAVE_ROUND -DLLDB_IN_LIBLLDB -D_GLIBCXX_USE_CXX11_ABI=1 -D_GNU_SOURCE -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS -I/home/zorgatone/source/repos/opensource/llvm-project/build/tools/lldb/source/API -I/home/zorgatone/source/repos/opensource/llvm-project/lldb/source/API -I/home/zorgatone/source/repos/opensource/llvm-project/lldb/include -I/home/zorgatone/source/repos/opensource/llvm-project/build/tools/lldb/include -I/home/zorgatone/source/repos/opensource/llvm-project/build/include -I/home/zorgatone/source/repos/opensource/llvm-project/llvm/include -I/usr/include/python3.13 -I/home/zorgatone/source/repos/opensource/llvm-project/llvm/../clang/include -I/home/zorgatone/source/repos/opensource/llvm-project/build/tools/lldb/../clang/include -I/home/zorgatone/source/repos/opensource/llvm-project/lldb/source -I/home/zorgatone/source/repos/opensource/llvm-project/build/tools/lldb/source -isystem /usr/include/libxml2 -fPIC -fno-semantic-interposition -fvisibility-inlines-hidden -Werror=date-time -Wall -Wextra -Wno-unused-parameter -Wwrite-strings -Wcast-qual -Wno-missing-field-initializers -pedantic -Wno-long-long -Wimplicit-fallthrough -Wno-maybe-uninitialized -Wno-nonnull -Wno-class-memaccess -Wno-dangling-reference -Wno-redundant-move -Wno-pessimizing-move -Wno-array-bounds -Wno-stringop-overread -Wno-noexcept-type -Wdelete-non-virtual-dtor -Wsuggest-override -Wno-comment -Wno-misleading-indentation -Wctad-maybe-unsupported -fdiagnostics-color -ffunction-sections -fdata-sections -Wno-unknown-pragmas -Wno-strict-aliasing -Wno-stringop-truncation -O3 -DNDEBUG -std=c++17 -fPIC  -fno-exceptions -funwind-tables -fno-rtti  -w -Wno-sequence-point -Wno-cast-qual -MD -MT tools/lldb/source/API/CMakeFiles/liblldb.dir/__/__/bindings/python/LLDBWrapPython.cpp.o -MF tools/lldb/source/API/CMakeFiles/liblldb.dir/__/__/bindings/python/LLDBWrapPython.cpp.o.d -o tools/lldb/source/API/CMakeFiles/liblldb.dir/__/__/bindings/python/LLDBWrapPython.cpp.o -c /home/zorgatone/source/repos/opensource/llvm-project/build/tools/lldb/bindings/python/LLDBWrapPython.cpp
/home/zorgatone/source/repos/opensource/llvm-project/build/tools/lldb/bindings/python/LLDBWrapPython.cpp: In function 'PyObject* SWIG_runtime_data_module()':
/home/zorgatone/source/repos/opensource/llvm-project/build/tools/lldb/bindings/python/LLDBWrapPython.cpp:1456:39: error: 'PyImport_AddModuleRef' was not declared in this scope; did you mean 'PyImport_AddModule'
 1456 |     Swig_runtime_data_module_global = PyImport_AddModuleRef(SWIG_RUNTIME_MODULE);
      |                                       ^~~~~~~~~~~~~~~~~~~~~
      |                                       PyImport_AddModule
/home/zorgatone/source/repos/opensource/llvm-project/build/tools/lldb/bindings/python/LLDBWrapPython.cpp: In function 'int SwigPyObject_Check(PyObject*)':
/home/zorgatone/source/repos/opensource/llvm-project/build/tools/lldb/bindings/python/LLDBWrapPython.cpp:1058:44: error: 'PyType_GetFullyQualifiedName' was not declared in this scope; did you mean 'WIG_PyType_GetFullyQualifiedName'
 1058 | # define SWIG_PyType_GetFullyQualifiedName PyType_GetFullyQualifiedName
      |                                            ^~~~~~~~~~~~~~~~~~~~~~~~~~~~
/home/zorgatone/source/repos/opensource/llvm-project/build/tools/lldb/bindings/python/LLDBWrapPython.cpp:2128:24: note: in expansion of macro 'WIG_PyType_GetFullyQualifiedName'
 2128 |     PyObject *tpname = SWIG_PyType_GetFullyQualifiedName(op_type);
      |                        ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/home/zorgatone/source/repos/opensource/llvm-project/build/tools/lldb/bindings/python/LLDBWrapPython.cpp: In function 'int SwigPyPacked_Check(PyObject*)':
/home/zorgatone/source/repos/opensource/llvm-project/build/tools/lldb/bindings/python/LLDBWrapPython.cpp:1058:44: error: 'PyType_GetFullyQualifiedName' was not declared in this scope; did you mean 'WIG_PyType_GetFullyQualifiedName'
 1058 | # define SWIG_PyType_GetFullyQualifiedName PyType_GetFullyQualifiedName
      |                                            ^~~~~~~~~~~~~~~~~~~~~~~~~~~~
/home/zorgatone/source/repos/opensource/llvm-project/build/tools/lldb/bindings/python/LLDBWrapPython.cpp:2545:24: note: in expansion of macro 'WIG_PyType_GetFullyQualifiedName'
 2545 |     PyObject *tpname = SWIG_PyType_GetFullyQualifiedName(op_type);
      |                        ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/home/zorgatone/source/repos/opensource/llvm-project/build/tools/lldb/bindings/python/LLDBWrapPython.cpp: In function 'wig_type_info* SWIG_Python_TypeQuery(const char*)':
/home/zorgatone/source/repos/opensource/llvm-project/build/tools/lldb/bindings/python/LLDBWrapPython.cpp:1115:39: error: 'PyDict_GetItemStringRef' was not declared in this scope; did you mean 'PyDict_GetItemString'
 1115 | # define SWIG_PyDict_GetItemStringRef PyDict_GetItemStringRef
      |                                       ^~~~~~~~~~~~~~~~~~~~~~~
/home/zorgatone/source/repos/opensource/llvm-project/build/tools/lldb/bindings/python/LLDBWrapPython.cpp:3278:3: note: in expansion of macro 'WIG_PyDict_GetItemStringRef'
 3278 |   SWIG_PyDict_GetItemStringRef(cache, type, &obj);
      |   ^~~~~~~~~~~~~~~~~~~~~~~~~~~~
[9/67] Building CXX object tools/lldb/tools/lldb-dap/CMakeFiles/lldbDAP.dir/OutputRedirector.cpp.o
/home/zorgatone/source/repos/opensource/llvm-project/lldb/tools/lldb-dap/OutputRedirector.cpp: In member function 'void lldb_dap::OutputRedirector::Stop()':
/home/zorgatone/source/repos/opensource/llvm-project/lldb/tools/lldb-dap/OutputRedirector.cpp:108:18: warning: ignoring return value of 'size_t write(int, const void*, size_t)' declared with attribute 'warn_unused_result' [-Wunused-result]
  108 |     (void)::write(fd, kCloseSentinel.data(), kCloseSentinel.size());
      |           ~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
[10/67] Building CXX object tools/lldb/tools/lldb-dap/CMakeFiles/lldbDAP.dir/ProgressEvent.cpp.o
[11/67] Building CXX object tools/lldb/tools/lldb-dap/CMakeFiles/lldbDAP.dir/EventHelper.cpp.o
[12/67] Building CXX object tools/lldb/tools/lldb-dap/CMakeFiles/lldbDAP.dir/ProtocolUtils.cpp.o
[13/67] Building CXX object tools/lldb/tools/lldb-dap/CMakeFiles/lldbDAP.dir/JSONUtils.cpp.o
ninja: build stopped: subcommand failed.
```

Not sure if it's enough information, my python version prints `Python 3.13.9`

--- a/lldb/cmake/modules/LLDBConfig.cmake
+++ b/lldb/cmake/modules/LLDBConfig.cmake
@@ -182,7 +182,12 @@ if (LLDB_ENABLE_PYTHON)
 
   # Enable targeting the Python Limited C API.
   set(PYTHON_LIMITED_API_MIN_SWIG_VERSION "4.2")
-  set(AFFECTED_BY_SWIG_BUG SWIG_VERSION VERSION_EQUAL "4.4.0" AND Python3_VERSION VERSION_GREATER_EQUAL "3.13")
+  if (SWIG_VERSION VERSION_EQUAL "4.4.0" AND Python3_VERSION VERSION_GREATER_EQUAL "3.13")
+    set(AFFECTED_BY_SWIG_BUG TRUE)
+  else()
+    set(AFFECTED_BY_SWIG_BUG FALSE)
+  endif()
+
   if (SWIG_VERSION VERSION_GREATER_EQUAL PYTHON_LIMITED_API_MIN_SWIG_VERSION
       AND NOT LLDB_EMBED_PYTHON_HOME AND NOT AFFECTED_BY_SWIG_BUG)
     set(default_enable_python_limited_api ON)
@@ -195,7 +200,7 @@ if (LLDB_ENABLE_PYTHON)
 
   # Diagnose unsupported configurations.
   if (LLDB_ENABLE_PYTHON_LIMITED_API AND AFFECTED_BY_SWIG_BUG)
-    message(SEND_ERROR "LLDB_ENABLE_PYTHON_LIMITED_API is not compatible with SWIG 4.4.0 and Python 3.13 due to a bug in SWIG: https://github.com/swig/swig/issues/3283")
+    message(SEND_ERROR "LLDB_ENABLE_PYTHON_LIMITED_API is not compatible with SWIG 4.4.0 and Python >= 3.13 due to a bug in SWIG: https://github.com/swig/swig/issues/3283")
   endif()
   if (LLDB_ENABLE_PYTHON_LIMITED_API AND LLDB_EMBED_PYTHON_HOME)
     message(SEND_ERROR "LLDB_ENABLE_PYTHON_LIMITED_API is not compatible with LLDB_EMBED_PYTHON_HOME")

